src/controllers/userController.ts

```diff
@@ -8,15 +8,8 @@ import { BadRequestError, UnauthorizedError } from '../errors/httpErrors';
 import { ACCESS_TOKEN_SECRET } from '../consts';
 
 async function register(req: Request, res: Response, next: NextFunction) {
```

Comment:
Just wanted to check if this means Front-End will need to include 'role' attribute in the req body or should we set a default value for the role (like 'User')  so they don't have to modify their current POST request

Reply:
Yeap, the default value for `role` attribute will be `user`. Can check `User.ts` to find out more. It is also set on the database side using migrations.